opencga-master/src/main/java/org/opencb/opencga/master/monitor/daemons/ExecutionDaemon.java

```diff
@@ -324,22 +332,46 @@ protected void checkJobs() throws CatalogException {
             }
             logger.info("----- EXECUTION DAEMON  ----- Organization={} --> pending={}, queued={}, running={}", organizationId, pendingJobs,
                     queuedJobs, runningJobs);
+            totalPendingJobs += pendingJobs;
+            totalQueuedJobs += queuedJobs;
+            totalRunningJobs += runningJobs;
         }
 
-        /*
-        PENDING JOBS
-        */
-        checkPendingJobs(organizationIds);
+        if (totalQueuedJobs == 0) {
+            // If there are no queued jobs, we can queue new jobs
+            List<Job> pendingJobs = new LinkedList<>();
+            List<Job> runningJobs = new LinkedList<>();
 
-        /*
-        QUEUED JOBS
-        */
-        checkQueuedJobs(organizationIds);
+            for (String organizationId : organizationIds) {
+                try (DBIterator<Job> iterator = jobManager.iteratorInOrganization(organizationId, pendingJobsQuery, queryOptions, token)) {
+                    while (iterator.hasNext()) {
+                        pendingJobs.add(iterator.next());
+                    }
+                } catch (Exception e) {
+                    logger.error("{}", e.getMessage(), e);
```

Comment:
Is it safe to ignore these exceptions?

Reply:
We are logging those exceptions. What else are we supposed to do there? The daemon shouldn't stop, right?